tests/mocks/errors.js

```diff
@@ -122,7 +122,7 @@ define([], function () {
     incorrectEmailCase: {
       status: 400,
       headers: {},
-      body: '{"code":400, "errno": 120}'
+      body: '{"code":400, "errno": 120, "email": "a@b.com"}'
```

Comment:
The "incorrect case" error only gets special treatment if it returns the correctly-cased email.

Reply:
👍